docs/styles/_search.scss

```diff
@@ -0,0 +1,3 @@
+.search-block {
```

Comment:
Are you sure if we render this component with other (i.e. as a header) it will still work?